docs/user/canonical-urls.rst

```diff
@@ -0,0 +1,55 @@
+Canonical URLs
+==============
+
+`Canonical URLs`_ allow you to specify the preferred version of a web page to prevent duplicated content.
+They are mainly used by search engines to link users to the correct
+version and domain of your documentation.
+
+.. _Canonical URLs: https://datatracker.ietf.org/doc/html/rfc6596
+
+.. contents:: Contents
+    :local:
+
+Example
+-------
+
+The canonical URL is generated based on:
+
+* The default version of your project (usually "latest" or "stable").
+* The canonical :doc:`custom domain </custom-domains>` if you have one,
+  otherwise the default :ref:`subdomain <hosting:subdomain support>` will be used.
+
+For example, if you have a project named ``docs``
+with a custom domain ``https://docs.example.com``.
+A search engine like Google will index both domains, ``http://docs.readthedocs.io`` and
+``https://docs.example.com``.
+
+But you'll want to use ``https://docs.example.com`` as your canonical domain,
+this means that when Google indexes a page like ``https://docs.readthedocs.io/en/9.0/``,
+it will know that it should really point at ``https://docs.example.com/en/latest/``.
+
```

Comment:
This feels really confusing to a user... I feel like we aren't explaining why this is useful here.